### DISCUSSION TITLE
Loading contributor plugin

### DISCUSSION THREAD

**Discussion by letitiayhho** [jspsych/jsPsych#3004] [Q&A]
Hi all, sorry if this is a dumb question but I'm trying to get the [libet-intentional-binding](https://github.com/jspsych/jspsych-contrib/tree/main/packages/plugin-libet-intentional-binding) plugin to work. The documentation says just to add the line below to index.html, but it doesn't seem to be working. My code is on GitLab and I'm hosting the experiment on Pavlovia if that helps.

`<script src="https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.2.0">`

**Comment by ChristopheBossens** [comment]
Hi,

What other code do you have in your script, and what is it exactly that is not working? I could get the demo code working with the following code: 

```
<!DOCTYPE html>
<html>
  <head>
    <title>My experiment</title>
    <script src="../jspsych/jspsych.js"></script>
    <script src="https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.2.0"></script>

    <link href="../jspsych/jspsych.css" rel="stylesheet" type="text/css" />
  </head>
  <body></body>
  <script>
    const jsPsych = initJsPsych();


    var trial = {
    type: jsPsychLibetIntentionalBinding,
    cond: 'operant-tone',
    tone_file: 'tone.mp3'
}

    jsPsych.run([trial]);
  </script>
</html>
```

**Reply by letitiayhho** [reply]
Below is my code, I think it's similar to yours, I just added code to make the script work on Pavlovia. I've attached a screenshot of the errors I'm getting.

```
<!DOCTYPE html>
<html>
  <head>
    <title>My experiment</title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="jspsych-6.1.0/jspsych.js"></script>
    <script src="https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.2.0"></script>
    <script type="text/javascript" src="lib/jspsych-pavlovia-3.0.0.js"></script>
    <link
      href="jspsych-6.1.0/css/jspsych.css"
      rel="stylesheet"
      type="text/css"
    />
  </head>
  <body></body>
  <script>

    const pavlovia_init = {
      type: "pavlovia",
      command: "init",
    };

    var trial = {
      type: jsPsychLibetIntentionalBinding,
      cond: 'operant-tone',
      tone_file: 'tone.mp3'
    };

    const pavlovia_finish = {
      type: "pavlovia",
      command: "finish",
    };

    let timeline = [];
    timeline.push(pavlovia_init);
    timeline.push(trial);
    timeline.push(pavlovia_finish);

    jsPsych.init({
      timeline: timeline,
      use_webaudio: false,
    });

  </script>
</html>

```

<img width="967" alt="Screen Shot 2023-04-14 at 11 29 47 AM" src="https://user-images.githubusercontent.com/58232998/232102967-5c1f20af-663b-4aa1-9038-1023298f1ebf.png">

**Reply by ChristopheBossens** [reply]
Can you try with the most recent version of jsPsych (7.3)?
